src/phpDocumentor/Parser/FlySystemFactory.php

```diff
@@ -16,15 +16,17 @@
 use Flyfinder\Finder;
 use InvalidArgumentException;
 use League\Flysystem\Adapter\Local;
+use League\Flysystem\AdapterInterface;
 use League\Flysystem\Filesystem;
 use League\Flysystem\FilesystemInterface;
 use League\Flysystem\MountManager;
 use LogicException;
 use phpDocumentor\Dsn;
 use const LOCK_EX;
 use function hash;
+use function in_array;
 
-final class FlySystemFactory implements FileSystemFactory
+class FlySystemFactory implements FileSystemFactory
```

Comment:
Because it is mocked in the TransformerTest; one of my upcoming goals is to go through the non-final classes and determine whether they really cannot be final and/or refactor them to be final. But in this case, functionality won :)